### PR TITLE
The Nimbus repository was renamed (fix broken links)

### PR DESCRIPTION
**Pull request by zah**

### CODE REVIEW

File: src/pages/Clients/Eth2/Nimbus.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ export const NimbusDetails = ({ shortened }: { shortened?: boolean }) => (
           <Link
             external
             primary
-            to="https://status-im.github.io/nim-beacon-chain/validator.html#key-management"
+            to="https://status-im.github.io/nimbus-eth2/validator.html#key-management"
```
Comment (hwwhww) [inline src/pages/Clients/Eth2/Nimbus.tsx:78]: @zah
Is this page WIP?
I saw

image

**Comment by sachayves** [comment]
@hwwhww @zah updated that link here
https://github.com/ethereum/eth2.0-deposit/pull/188

**Comment by zah** [comment]
To make things easier, I've also updated the link in this PR (so #188 can be closed).

**Review by hwwhww — APPROVED** [review]
lgtm, thanks!

**Pull request merged**